Services/CO.CDP.OrganisationInformation.Persistence/CO.CDP.OrganisationInformation.Persistence.csproj

```diff
@@ -18,4 +18,9 @@
         <Folder Include="Migrations\" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Compile Remove="Migrations\20240818202741_CreatePersonInvite.cs" />
+      <Compile Remove="Migrations\20240818202741_CreatePersonInvite.Designer.cs" />
+    </ItemGroup>
+
```

Comment:
```suggestion
```

Reply:
@dharmverma So we don't actually need it? I can just remove as above?

Reply:
yes